<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modification Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(47, 47, 47);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(213, 213, 213);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>820</y>
      <width>239</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="SettingBox">
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="volumeBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Volume</string>
    </property>
    <widget class="QSlider" name="volumeSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>22</width>
       <height>160</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-12</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>3</number>
     </property>
    </widget>
    <widget class="QPushButton" name="volumeButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="trackBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>1171</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>Track</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="modificationBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>1011</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Modify</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>431</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QComboBox" name="unitComboBox">
        <item>
         <property name="text">
          <string>seconds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>samples</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="insertButton">
        <property name="text">
         <string>Insert</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="toSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Unit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="fromSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="trimButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>Trim</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>491</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string>Modification instruction

 1. Trim - trims track from given value to given value
 2. Inserts - inserts in given moment (stop value is ignored) given signal
 3. Remove - removes given part of track 

After pushing correct button changes will be shown in Track view.
If you want to update track in main window, click Apply in the right down corner
of this window.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menuExport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
